optimizations/index.js

```diff
@@ -1,4 +1,4 @@
-const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
+const TerserPlugin = require('uglifyjs-webpack-plugin');
```

Comment:
Shouldn't this be `require('terser-webpack-plugin')`?

Reply:
omg